core/src/main/scala/org/apache/spark/ui/exec/ExecutorThreadDumpPage.scala

```diff
@@ -76,7 +76,12 @@ private[ui] class ExecutorThreadDumpPage(
         <div class="bs-example" data-example-id="simple-form-inline">
           <div class="form-group">
             <div class="input-group">
-              Search: <input type="text" class="form-control" id="search" oninput="onSearchStringChange()"></input>
+              <span data-toggle="tooltip" data-placement="top" title="Searches (case insensitive) for
```

Comment:
Hm, I still think this might be noisy. Too many tooltips will start to interfere with usability of the screen. I think it's pretty obvious what search does, so I'm not sure this clarifies much.

Reply:
@srowen  i have placed tooltip only for locks. could you check